### PR TITLE
Update Namesilo.php

### PR DESCRIPTION
**Pull request by lukapaunovic**
I removed throwing exception when someone is not using Payment ID, this prevented them from using module without Payment ID defined.
But there is an issue, this is how input box is showing. 

`<input id="el-Payment_ID" type="Payment_ID" name="config[Payment_ID]" value="" required="required">`

It is set to REQUIRED and I have no idea how do i change that... I'm going to contact boxbilling musicboxlimited for this and i'm gonna do another pull request.
So basically, once you define Payment ID only way to remove it is to inspect element and remove `required="required"` and then save it.

### CODE REVIEW

**Comment by lukapaunovic** [comment]
If you don't implement this ASAP many users (if not already) are experiencing issues using module without Payment ID.

**Comment by lukapaunovic** [comment]
Are you going to accept my pull request omg
